contracts/test/ocr2/aggregator.test.ts

```diff
@@ -241,7 +244,7 @@ describe('aggregator.cairo', function () {
     assert.equal(e.data.round_id, 1n)
     assert.equal(e.data.observation_timestamp, 1n)
     assert.equal(e.data.epoch_and_round, 1n)
-    assert.equal(e.data.reimbursement, 0n)
+    // assert.equal(e.data.reimbursement, 0n)
```

Comment:
Let's test a number here

Reply:
This is based on the gas calculation so we'd have to update any time the constant changes